src/sentry/integrations/github/client.py

```diff
@@ -224,6 +224,25 @@ def get_commit(self, repo: str, sha: str) -> Any:
         """
         return self.get_cached(f"/repos/{repo}/commits/{sha}")
 
+    def get_merge_commit_sha_from_commit(self, repo: str, sha: str) -> str | None:
+        """
+        Get the merge commit sha from a commit sha.
+        """
+        response = self.get_pullrequest_from_commit(repo, sha)
+        if not response or (isinstance(response, list) and len(response) != 1):
+            # the response should return a single merged PR, return if multiple
+            return None
+
+        pull_request = response[0]
```

Comment:
Nit: I like the idiom `(pull_request,) = response` because it communicates the expectation that the length is 1 and raises otherwise.